.github/workflows/gh-pages-deploy.yaml

```diff
@@ -26,7 +26,12 @@ jobs:
         run: dart pub get
 
       - name: Generate documentation files
-        run: dart run bin/generate_web_docs_content.dart -p lib/lints -o doc/docusaurus/docs -r README.md -s
+        run: >
+          dart run bin/generate_web_docs_content.dart
+            --path lib/lints
+            --docs-dir doc/docusaurus/docs/generated
+            --readme README.md
+            --sort-rules
```

Comment:
@solid-daniilmarchenko it's usually better to use full arg names for CI and such -- easier to read and comprehend.

Reply:
got it, thanks for advice

Reply:
i've decided to remove --sort-rules parameter btw cause it does not makes any sense while generating files to the folder